back/app/controllers/web_api/v1/admin_publications_controller.rb

```diff
@@ -7,6 +7,17 @@ class WebApi::V1::AdminPublicationsController < ::ApplicationController
   def index
     publication_filterer = AdminPublicationsFilteringService.new
     publications = policy_scope(AdminPublication.includes(:parent))
+
+    if params[:search].present?
+      Rails.logger.info("*" * 30)
```

Comment:
Rubocop doesn't like these double quotes, in case you wondered why linting is failing.

Reply:
Yeah, it showed up in my IDE, but since I have to delete this before merge I just left it fo rnow